run/john.conf

```diff
@@ -363,11 +363,11 @@ Frequency = 141
 
 [ZTEX:sha512crypt]
 #TargetRounds = 5000
-# Design tools reported possible frequency to be 225.1 MHz.
+# Design tools reported possible frequency to be 215 MHz.
 # We never encountered a board where this worked anywhere close
-# to such high frequency. Default frequency is set to 135 MHz.
-# Some lucky boards run this OK at 160 MHz, but usually not more.
-#Frequency = 135
+# to such high frequency. Default frequency is set to 160 MHz.
+# Some lucky boards might run at some higher frequency.
+Frequency = 160
```

Comment:
We need to similarly change the default frequency in the section below for `drupal7-ztex`, and also change both default frequencies in the `.c` files.